spec/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.Name = nil
+    expect(subject).to_not be_valid
+  end
+
```

Comment:
- [ ] I think it would be great if you test the validation of PostsCounter must be an integer greater than or equal to zero so kindly add test to this validation as well. it could be somthing like this 🔥✅✨

```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```